.github/workflows/rust.yml

```diff
@@ -71,6 +65,31 @@ jobs:
           # we run tests using BitVec<u64,_> which doesn't.
           args: --all-features --target wasm32-unknown-unknown
 
+  no_std:
+    name: Check no_std build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      # Set default target aarch64-unknown-none because it doesn't support `std`.
+      # So this will fail if the crate is not no_std compatible.
+      - name: Rust create rust-toolchain.toml
+        run: echo -e "[toolchain]\nchannel = \"nightly\"\ntargets = [\"aarch64-unknown-none\"]\nprofile = \"minimal\"" > rust-toolchain.toml
+
+      # no_std version currently needs nightly due to error_in_core feature
+      - name: Install Rust nightly toolchain
+        run: rustup show
```

Comment:
Can we just use the `actions-rs/toolchain@v1` instead like the above step for consistency :)

Reply:
Removed the rustup show cmd but I left the rust-toolchain.toml. To my understanding, `override` does only set the toolchain, not the target (https://rust-lang.github.io/rustup/overrides.html#directory-overrides). 

Copying from the PR description for better visibility: I experienced some troubles here because specifying  `--target` when running cargo check seems to automatically activate dev-dependencies (which have std deps). I wasn't sure if leaving away --target still uses the no-std environment target. So to be sure, CI now creates a rust-toolchain.toml that specifies the target.